### PR TITLE
report: fix flicker in normal case report

### PR DESCRIPTION
**Pull request by yeonjuan**
**Summary**

Bugfix

change `.lh-sticky-header`'s position `sticky` to `fixed`.

**Related Issues/PRs**

#9141

### CODE REVIEW

**Comment by connorjclark** [comment]
This needs to be sticky for devtools (#9023). Could you add a sticky declaration for `.lh-sticky-header.lh-devtools`, along with a comment? like:

```
/* The report within DevTools is placed in a container with overflow, which changes the placement of this header unless we change `position` to `fixed.` */
```

So we can get this nice fix for the regular report, but further work will be needed to fix this stutter in DevTools.

___

For why this is different in DevTools: the report is within a containing element. Normally, the report is the entire document, so `fixed` works fine in the normal case.

https://stackoverflow.com/a/48355127/2788187
> If it is defined inside some container, it is positioned with respect to that container. If the container has some overflow(scroll), depending on the scroll offset it turns into position:fixed.

**Comment by yeonjuan** [comment]
@connorjclark Thanks! i changed it. :)

File: lighthouse-core/report/html/templates.html [hunk 1]
```diff
@@ -185,6 +185,11 @@
       pointer-events: none;
     }
 
+    .lh-sticky-header.lh-devtools {
```
Comment (connorjclark) [inline lighthouse-core/report/html/templates.html]:
```suggestion
    .lh-devtools .lh-sticky-header {
```

File: lighthouse-core/report/html/templates.html [hunk 2]
```diff
@@ -185,6 +185,11 @@
...
+      /* The report within DevTools is placed in a container with overflow, which changes the placement of this header unless we change `position` to `fixed.` */
```
Comment (connorjclark) [inline lighthouse-core/report/html/templates.html]:
```suggestion
      /* The report within DevTools is placed in a container with overflow, which changes the placement of this header unless we change `position` to `sticky.` */
```

**Pull request merged**